window-purpose-core.el

```diff
@@ -168,7 +170,7 @@ If no purpose was determined, return `default-purpose'."
 
 (defun purpose-buffers-with-purpose (purpose)
   "Return a list of all existing buffers with purpose PURPOSE."
-  (cl-remove-if-not #'(lambda (buffer)
+  (cl-delete-if-not #'(lambda (buffer)
```

Comment:
I think this is ok, but will have to test later that it doesn't modify the internal buffer list.

Reply:
As per https://www.gnu.org/software/emacs/manual/html_node/elisp/Buffer-List.html
>The list returned by buffer-list is constructed specifically; it is not an internal Emacs data structure, and modifying it has no effect on the order of buffers.

So this is fine. Just a few bytes though, so I'm not sweating it at this point (I was trying to save every cons I could when it was being called many many times)